AzureFunctions.v3.Durable/Orchestrator.cs

```diff
@@ -24,7 +27,7 @@ public static async System.Threading.Tasks.Task<HttpResponseMessage> RunAsync([H
             if (eventGridSoleItem.EventType == @"Microsoft.EventGrid.SubscriptionValidationEvent")
```

Comment:
We should follow the pattern here - https://docs.microsoft.com/en-us/azure/event-grid/receive-events, i.e. use TryGetSystemEventData and use the strongly typed SubscriptionResponse model.